BinaryEncoding.md

```diff
@@ -81,7 +81,7 @@ Yes:
   - A table (sorted by offset) containing, for each type which has opcodes:
     + A standardized string literal [type name](AstSemantics.md#local-and-memory-types).
       The index of a type name in this table is referred to as a type ID
-    + 32-bit offset of its opcode table within the section
+    + 64-bit offset of its opcode table within the section
```

Comment:
I'm sorry I even mentioned it. We're probably talking single digit number of bits for compressed binary. This is fine.